language/evm/move-to-yul/src/yul_functions.rs

```diff
@@ -152,7 +154,8 @@ AbortBuiltin: "() {
 //   arena style.
 Malloc: "(size) -> offs {
     offs := mload(${MEM_SIZE_LOC})
-    mstore(${MEM_SIZE_LOC}, add(offs, size))
+    // pad to word size
+    mstore(${MEM_SIZE_LOC}, add(offs, shr(add(size, 31), 5)))
```

Comment:
Here I suppose we are padding `size` to a multiple of 32. If so, did we miss a `shl` by 5? Should this line be the following instead:
```
mstore(${MEM_SIZE_LOC}, add(offs, shl(shr(add(size, 31), 5),5)))
```

Reply:
Fixed in PR #79  (also the wrong order of operands to shift functions)